Source/JavaScriptCore/disassembler/ARM64/A64DOpcode.cpp

```diff
@@ -129,10 +129,13 @@ static const OpcodeGroupInitializer opcodeGroupList[] = {
     OPCODE_GROUP_ENTRY(0x1e, A64DOpcodeFloatingPointIntegerConversions),
 };
 
+Lock A64DOpcode::s_lock;
 bool A64DOpcode::s_initialized = false;
```

Comment:
If we use std::call_once, this s_initialized is not necessary.

Reply:
Good call! Changed in latest push.